docs/earth-command/earth-command.md

```diff
@@ -32,9 +30,8 @@
         [--push] [--no-cache] [--allow-privileged|-P]
         [--ssh-auth-sock <path-to-sock>]
         [--git-username <git-user>] [--git-password <git-pass>]
-        [--git-url-instead-of <git-instead-of>]
         [--buildkit-host <bk-host>] [--buildkit-cache-size-mb <cache-size-mb>]
-        [--buildkit-image <bk-image>] [--no-loop-device]
+        [--no-loop-device]
```

Comment:
Probably worth removing all the other options we plan not to support here, so that the command looks cleaner. Just mentioning them below (that they have moved) should be enough.